ci-cd-bootstrap/params/nexus/build_params

```diff
@@ -0,0 +1,2 @@
+VOLUME_CAPACITY=10Gi
```

Comment:
These aren't really "build" params, so I'd recommend naming the file something a bit more descriptive - i.e.:
```
ci-cd-bootstrap/params/nexus/deployment/<stage>/params
```

Reply:
Fixed